packages/ui/src/molecules/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, fireEvent } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import Alert from './Alert'
+
+describe('Alert', () => {
+  describe('Data attributes', () => {
+    it('should have `data-store-alert` attribute', () => {
+      const { getByTestId } = render(<Alert>Testing</Alert>)
+
+      expect(getByTestId('store-alert')).toHaveAttribute('data-store-alert')
+    })
+
+    it('should have `data-store-alert-button` attribute when dismissible', () => {
+      const { getByTestId } = render(<Alert dismissible>Testing</Alert>)
+
+      expect(getByTestId('store-alert-button')).toHaveAttribute(
+        'data-store-alert-button'
+      )
+    })
+  })
+
+  it('should render icon when the icon prop is present', () => {
+    const { getByTestId } = render(
+      <Alert icon={<div data-testid="icon">foo</div>}>Testing</Alert>
+    )
+
+    expect(getByTestId('icon')).toBeInTheDocument()
+  })
+
+  describe('User actions', () => {
```

Comment:
What do you think about adding a test to check if the user clicks inside the alert won't trigger the onClose, only if the user clicks inside the button? You test the behavior described above and related to the `event.stopPropagation` inside the handle close callback

Reply:
Nice, I'm going to implement it.